### PR TITLE
TD-28129: add the cloud docs to tools for yonghong and powerbi

### PR DESCRIPTION
**Pull request by wingwing2005**

### CODE REVIEW

File: docs/en/09-tools/07-powerbi.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: PowerBI
+title: Use PowerBI
+description: This document describes how to integrate TDengine Cloud with Microsoft PowerBI for data visualization.
+---
+
+Power BI is a business analytics tool provided by Microsoft. With TDengine ODBC driver, PowerBI can access time series data stored in the TDengine Cloud instance. You can import tag data, original time series data, or aggregated data into Power BI from a TDengine Cloud instance, to create reports or dashboard without any coding effort. The minimize TDengine version is 3.2.2.0.
```
Comment (danielclow) [inline docs/en/09-tools/07-powerbi.md:7]: Should delete "The minimize TDengine version is 3.2.2.0." for cloud docs

File: docs/en/09-tools/07-powerbi.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Prerequisite
+
+Power BI Desktop has been installed and running. (If not, please download and install latest Windows X64 version from [PowerBI](https://www.microsoft.com/download/details.aspx?id=58494).)
+
+## Install ODBC connector
+
+1. Only support Windows operation system. And you need to install [VC Runtime Library](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170) first. If already installed, please ignore this step.
+2. Install TDengine Windows client installation package（[Windows](https://www.tdengine.com/assets-download/3.0/TDengine-enterprise-client-3.2.2.0-Windows-x64.exe)）.
```
Comment (danielclow) [inline docs/en/09-tools/07-powerbi.md]: Can the link be automatically updated to the latest client version? (i.e. when 3.2.3.0 is released, the current link will become outdated)

File: docs/en/09-tools/07-powerbi.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+3. If you want to input some specific SQL, click "Advanced Options", and input your SQL in the open dialogue box and load the data.
+
+To better use Power BI to analyze the data stored in TDengine, you need to understand the concepts of dimention, metric, time serie, correlation, and use your own SQL to import data:
+
+1. Dimention: it's normally category (text) data to describe such information as device, collection point, model. In the supertable template of TDengine, we use tag columns to store the dimention information. You can use SQL like select distinct tbname, tag1, tag2 from supertable to get dimentions.
+2. Metric: quantitive (numeric) fileds that can be calculated, like SUM, AVERAGE, MINIMUM. If the collecting frequency is 1 second, then there are 31,536,000 records in one year, it will be too low efficient to import so big data into Power BI. In TDengine, you can use data partition query, window partition query, in combination with pseudo columns related to window, to import downsampled data into Power BI. For more details, please refer to [TDengine Specialized Queries](https://docs.tdengine.com/taos-sql/distinguished/).
```
Comment (danielclow) [inline docs/en/09-tools/07-powerbi.md]: Update link to https://docs.tdengine.com/cloud/taos-sql/distinguished/

File: docs/en/09-tools/07-powerbi.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+4. Data partition query: If you want to get the aggregate value of a lot of thermal meters, you can first partition the data and then perform a series of calculation in the partitioned data spaces. The SQL you need to use is partition by part_list. The most common of data partition usage is that when querying a supertable, you can partition data by subtable according to tags to form the data of each subtable into a single time serie to facilitate analytical processing of time series data.
+5. Time Serie: When curve plotting or aggregating data based on time lines, date is normally required. Data or time can be imported from Excel, or retrieved from TDengine using SQL statement like select _wstart date, count(*) cnt from test.meters where ts between A and B interval(1d) fill(0), in which the fill() subclause indicates the fill mode when there is data missing, pseudo column \_wstart indicates the date to retrieve.
+6. Correlation: Indicates how to correlate data. Dimentions and Metrics can be correlated by tbname, dates and metrics can be correlated by date. All these can cooperate to form visual reports.
+
+## 智能电表样例
```
Comment (danielclow) [inline docs/en/09-tools/07-powerbi.md]: Use English text, I guess "Smart meters example"?

File: docs/en/09-tools/07-powerbi.md (hunk 1 above)
Comment (wingwing2005) [reply docs/en/09-tools/07-powerbi.md:7]: We have displayed TDengine version in the Cloud, so if the TDengine version is less than 3.2.2.0, we will hide the tool.

File: docs/en/09-tools/07-powerbi.md (hunk 3 above)
Comment (wingwing2005) [reply docs/en/09-tools/07-powerbi.md]: OK, please refresh the latest pull request. I have updated the content.

**Pull request merged**